src/kinodynamic-path.cc

```diff
@@ -139,7 +139,8 @@ namespace hpp {
       if (timeRange ().second == 0)
         u = 0;
 
-      pinocchio::interpolate (device_,initial_, end_, u, result);
+      pinocchio::interpolate (device_,initial_.head (configSize),
+                              end_.head (configSize), u, result);
```

Comment:
Shouldn't it be `result.head (configSize)` instead of `result`

Reply:
`pinocchio::interpolate` writes the result at the beginning of vector `result`. The size of result is not expicitely checked.